### PR TITLE
Update default sizes for new dash width

### PR DESCRIPTION
**Pull request by kdoh**
Updates default sizes when adding cards to the dashboard to reflect the updated dashboard width.

- Completes Milestone 2: https://github.com/metabase/metabase/issues/36358

### CODE REVIEW

File: frontend/src/metabase/visualizations/shared/utils/sizes.ts [hunk 1]
```diff
@@ -43,23 +43,23 @@ const VISUALIZATION_SIZES: {
   },
   pie: {
     min: { width: 4, height: 3 },
-    default: { width: 8, height: 6 },
+    default: { width: 12, height: 8 },
   },
   funnel: {
     min: { width: 4, height: 3 },
     default: { width: 8, height: 6 },
```
Comment (cdeweyx) [inline frontend/src/metabase/visualizations/shared/utils/sizes.ts:50]: Any reason why funnels are still 8 default? That's the only one here that I was wondering, why not bump that up to 12 as well.

**Review by cdeweyx — APPROVED** [review]
Left one question on funnels but overall much much better

File: frontend/src/metabase/visualizations/shared/utils/sizes.ts (hunk 1 above)
Comment (kdoh) [reply frontend/src/metabase/visualizations/shared/utils/sizes.ts]: Nope, just forgot. Thanks for the reminder.

**Review by JesseSDevaney — APPROVED** [review]

**Comment by JesseSDevaney** [comment]
Waiting to merge until Milestone 1 is merged
- https://github.com/metabase/metabase/pull/38377

File: frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.tsx [hunk 2]
```diff
@@ -413,7 +413,10 @@ class DashboardHeaderContainer extends Component<DashboardHeaderProps> {
       // Add link card button
       buttons.push(
         <DashboardButtonTooltip key="add-link-card" label={t`Add link card`}>
-          <DashboardHeaderButton onClick={() => this.onAddLinkCard()}>
+          <DashboardHeaderButton
+            aria-label={t`Add link card`}
+            onClick={() => this.onAddLinkCard()}
+          >
```
Comment (kulyk) [inline frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.tsx:419]: ❤️

**Review by kulyk — APPROVED** [review]

**Comment by perivamsi** [comment]
force merging it because of a failing unit test that is [unrelated to this change](https://github.com/metabase/metabase/pull/38433)

**Pull request merged**